subsys/bluetooth/host/l2cap.c

```diff
@@ -43,7 +43,7 @@ LOG_MODULE_REGISTER(bt_l2cap, CONFIG_BT_L2CAP_LOG_LEVEL);
 #define L2CAP_LE_MIN_MTU		23
 #define L2CAP_ECRED_MIN_MTU		64
 
-#define L2CAP_LE_MAX_CREDITS		(CONFIG_BT_BUF_ACL_RX_COUNT - 1)
+#define L2CAP_LE_MAX_CREDITS		(BT_BUF_ACL_RX_COUNT - 1)
```

Comment:
Ditto here
```suggestion
#define L2CAP_LE_MAX_CREDITS		(CONFIG_BT_MAX_CONN)
```

Reply:
I can do it in a following PR if it's fine for both of you.

Reply:
Fine by me